### DISCUSSION TITLE
can't find dipy_buan_profiles!!!

### DISCUSSION THREAD

**Discussion by ZhengRuixi** [dipy/dipy#3278] [Tractography / Tractometry] [answered]
I want to find the detailed content of "dipy_buan_profiles" and learn the code of it, can I get it? Thank you

**Comment by ZhengRuixi** [comment]
dipy_buan_lmm "bundle_profiles/*" --out_dir "lmm_plots"
The commandline above can successfully run by dipy, but I still want to konw the running process and the exact code, thank you!

**Reply by skoudoro** [reply, marked as answer]
Hi @ZhengRuixi,

here you are: https://github.com/dipy/dipy/blob/0799aeba4c418847aa8eb6e80cc7fafd6a8c06f2/dipy/workflows/stats.py#L178.

you can find more with a search in DIPY repository
